@xen-orchestra/web-core/lib/locales/fr.json

```
@@ -465,17 +495,23 @@
   "vcpus-used": "vCPUs utilisés",
   "vdis": "VDI | VDI | VDIs",
   "version": "Version",
+  "Video-RAM": "memoire video",
   "vif": "VIF",
   "vif-device": "VIF #{device}",
   "vif-status": "Statut du VIF",
   "vifs": "VIFs",
+  "Viridian": "Viridian",
+  "Virtual-TPM-(VTPM)": "TPM virtuel (vTPM)",
+  "Virtualization-boot-settings": "Virtualisation & paramètres de démarrage",
+  "Virtualization-mode": "Mode de virtualisation",
   "vlan": "VLAN",
   "vm": "VM",
   "vm-description": "Description de la VM",
   "vm-is-running": "La VM est en cours d'exécution",
+  "Vm-limit-topologie": "Limite de topologie physique des VM",
```

```suggestion
  "vm-limit-topology": "Limite de topologie physique des VMs",
```

I'm not sure about this trad